Makefile

```diff
@@ -13,7 +13,7 @@ test: ## Run all tests.
 # https://github.com/xd009642/tarpaulin/issues/161
 # https://github.com/xd009642/tarpaulin/issues/190#issuecomment-473564880
 cov: ## Run code coverage.
-	RUSTC="$$(pwd)/devtools/cov/rustc-proptest-fix" taskset -c 0 cargo tarpaulin --timeout 300 --exclude-files protocol/src/protocol_generated* test/* */tests/ --all -v --out Xml
+	RUSTC="$$(pwd)/devtools/cov/rustc-proptest-fix" taskset -c 0 cargo tarpaulin --timeout 300 --exclude-files */generated/ test/* */tests/ --all -v --out Xml
```

Comment:
It's better to quote the wildcard patterns. In some shell, they are expanded to nothing when there are no files matching the pattern. For example, in my zsh, `*/generated/` and `*/tests/` are all just removed from the command line:

```
$ ls -ld test/*
-rw-r--r--  1 ian  staff  191958 Aug 25 12:19 test/Cargo.lock
-rw-r--r--  1 ian  staff    1069 Aug 26 11:11 test/Cargo.toml
-rw-r--r--  1 ian  staff     622 Jul 27 12:34 test/README.md
-rw-r--r--  1 ian  staff    1873 Aug 25 13:46 test/integration.toml
drwxr-xr-x  9 ian  staff     288 Aug 25 13:46 test/src
lrwxr-xr-x  1 ian  staff      10 Aug 24 14:23 test/target -> ../target/

$ ls -ld */generated/
drwxr-xr-x  59 ian  staff  1888 Aug 26 11:14 .

$ ls -ld */tests/
drwxr-xr-x  59 ian  staff  1888 Aug 26 11:14 .
```

The latter two are the same as running `ls -ld`.

Reply:
updated